client/src/pages/EditOrganisationProfile.js

```diff
@@ -179,7 +179,7 @@ function EditOrganisationProfile(props) {
         );
       }
     })();
-  }, [orgProfileDispatch, organisationId]);
+  }, [orgProfileDispatch, organisationId, t]);
```

Comment:
Let's change all the `t`s to something more descriptive.

Reply:
For `const { t } = useTranslation();` I think just the `t`  okay.